src/core/core.datasetController.js

```diff
@@ -339,8 +339,7 @@ helpers.extend(DatasetController.prototype, {
 		var me = this;
 		var type = me.dataElementType;
 		return type && new type({
-			_ctx: me.chart.ctx,
-			_parsed: {}
+			_ctx: me.chart.ctx
```

Comment:
Looks like `createMetaDataset` and `createMetaData` could be replaced with `createMeta(type)`